.github/quarkus-github-bot.yml

```diff
@@ -407,6 +411,11 @@ triage:
       labels: [area/mandrel]
       titleBody: "mandrel"
       notify: [galderz, zakkak, Karm]
+    - id: native-image
+      labels: [area/native-image]
+      expression: |
+        matches("native", titleBody) && !matches("knative", titleBody
```

Comment:
I would use `title: "\\bnative\\b"` instead (it's missing a closing parenthesis anyway so it need fixing).